hostprocess/csi-proxy/build.sh

```diff
@@ -60,12 +53,6 @@ fi
 # Create manifest
 docker manifest create ${repository}/csi-proxy:${version} $manifest_entries
 
-# Annotate manifests
-for win_ver in "${win_vers[@]}"; do
```

Comment:
@marosset  do we still have an issue in containerd where this won't pull properly with out the right manifest or did that get resolved?